ghwatch.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python3
+'''
+Watch sorted github notifications from the terminal.
+'''
+import argparse
+from collections import namedtuple
+import datetime
+import json
+import os
+import re
+import shutil
+import sys
+import subprocess
+
+from github import Github, GithubObject
+
+from ghmeta import GhMeta
+from termlib.input import Key, clean_quit
+from termlib.tableprinter import Column, TablePrinter
+from termlib.attr import Attr
+
+DEFAULT_CONFIG = {
+    'ghbase': 'https://github.com/',
+    'ghtoken': '',
+    'browser': ['firefox', '--new-tab'],
+
+    # Repository with github metadata mirror (to get label data)
+    'meta': {'bitcoin/bitcoin': '/path/to/bitcoin-gh-meta'},
+
+    # Label priorities; the higher in this list, the higher the priority.
+    # When a PR or issue has multiple labels, the one with the highest priority will be
+    # shown. This is pretty arbitary, roughly going from specific to aspecific,
+    # and not a value judgement with regard to importance of components.
+    'label_prio': {'bitcoin/bitcoin': [
+      'Consensus',
+      'Mining',
+      'Mempool',
+      'TX fees and policy',
+      'UTXO Db and Indexes',
+      'Validation',
+      'P2P',
+      'Wallet',
+      'RPC/REST/ZMQ',
+      'Build system',
+      'Scripts and tools',
+      'Settings',
+      'Utils/log/libs',
+      'Tests',
+      'GUI',
+      'Docs',
+      'Descriptors',
+      'PSBT',
+      'Privacy',
+      'Resource usage',
+      'Block storage',
+      'Data corruption',
+      'Interfaces',
+      'Refactoring',
+    ]},
```

Comment:
Sort of unfortunate that the labels are hardcoded in the source code, but I do the same for my scripts (and let them fail once a label is modified/added/removed).

Reply:
Mind that this is only the default configuration. It's used to generate the default `ghwatch.json`, you can customize it afterward—even for non-bitcoin repos, although that is untested.